extension/package.json

```diff
@@ -17,7 +17,7 @@
     "check": "npm run lint && prettier --check .",
     "doctoc": "doctoc . --github",
     "format": "prettier --write .",
-    "zip-google-function": "cp index.googleFunction.js index.js && zip -r extension-module.zip src resources index.js package.json && rm index.js",
+    "zip-google-function": "cp index.googleFunction.js index.js && zip -r extension-module.zip src resources .extensionrc index.js package.json && rm index.js",
```

Comment:
Is it ok if `.extensionrc` is not present?

Reply:
yep I tested it.